src/components/molecules/Slider/style.scss

```diff
@@ -0,0 +1,69 @@
+.slider {
+    width: 100%;
+    height: auto;
+    overflow: hidden;
+    box-sizing: border-box;
+    display: flex;
+    align-items: center;
+
+    &__slide {
+        position: relative;
+        min-width: 100%;
+        height: auto;
+        overflow: hidden;
+
+        &__img {
+            width: 100%;
+        }
+    }
+
+    &__slide0-transitionOn {
+        @extend .slider__slide;
+        transform: translateX(0%);
+        transition: 0.3s;
+    }
+
+    &__slide-100-transitionOn {
```

Comment:
please leave only -transitionOn and Off classes and all translateX move to react component

Reply:
Changed